### ISSUE TITLE
.remove() not removing all tags

### ISSUE THREAD

**Issue by mafuchi** [hMatoba/Piexif#88]
Using the .remove() function and the output still has some exif info, like the creator of the IMG and tags.  The camera info is successfully removed, but personal information still seems to stay with the img. I used PIL and Piexif and the results show a noted difference. 
**Piexif**:
![Screenshot (151)](https://user-images.githubusercontent.com/6242892/55987331-2b9a8300-5c56-11e9-9286-4cc2450808e4.png)
**PIL**:
![Screenshot (150)](https://user-images.githubusercontent.com/6242892/55987018-7962bb80-5c55-11e9-88cc-90efb1fe4a0d.png)

Let me know if I am missing somehitng, or if you would require more info for this issue

**Comment by mafuchi** [comment]
Forgot to mention:
Piexif 1.1.2
Python 3.7.2
Windows 10 Pro

**Comment by pointerish** [comment]
Just out of curiosity. Is the image a JPG file? What is the filetype?

**Comment by mafuchi** [comment]
Yes, this was either a JPEG or JPG.  Still no idea why this happens.

**Comment by jiteshtandel** [comment]
I want to replace my Node JS function with Python using this library on AWS Lamda but I am also facing this same issue. Please give me any alternative solution. Thank you in advance.

**Comment by hMatoba** [comment]
Is it really exif? Isn't it XMP?
Does it allow exif specification? Doesn't it have multiple exif segmentation?

**Comment by jiteshtandel** [comment]
I want to remove XMP and EXIF tags. Basically, I want to remove all tags.

**Comment by farin** [comment]
I would be great if piexif can optionally remove XMP too.
